### ISSUE TITLE
Can't use / between tez and tez

### ISSUE THREAD

**Issue by catsigma** [OCamlPro/liquidity#51]
Using the latest next branch
```
Error: Type error:  Bad 2 args for primitive "/":
    tez
    tez
```
But Michelson support this operation between tez and tez.

**Comment by catsigma** [comment]
@mebsout This operation is needed in many situations. Please fix this. Thanks.

**Comment by mebsout** [comment]
Thanks for bringing it to our attention, this is now fixed (7b3aa987) in branch `next`
